scripts/update8.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 #
 # ===========================================================================
-# (c) Copyright IBM Corp. 2017 All Rights Reserved
+# (c) Copyright IBM Corp. 2018 All Rights Reserved
```

Comment:
Copyright should be `2017, 2018`. The first year indicated the year of the initial authoring, the second year indicates the last year it was changed.